.github/ISSUE_TEMPLATE/new_score.md

```diff
@@ -16,7 +16,6 @@ A new score or metric should be developed on a separate feature branch, rebased
     A reference to the paper which described the metrics, added to the API documentation
     For metrics which do not have a paper reference, an online source or reference should be provided
     For metrics which are still under development or which have not yet had an academic publication, they will be placed in a holding area within the API until the method has been properly published and peer reviewed (i.e. scores.emerging). The 'emerging' area of the API is subject to rapid change, still of sufficient community interest to include, similar to a 'preprint' of a score or metric.
-    Add your score to summary_table_of_scores.md in the documentation
```

Comment:
Should `summary_table_of_scores.md` be removed in this pull request?

Reply:
Yes, I really thought I did that step, I've done it now.